docs/pages/workflow/customizing.mdx

```diff
@@ -3,73 +3,92 @@ title: Add custom native code
 
-If you manually modify the **android** and **ios** directories, you must set up new libraries manually. Running `npx expo prebuild` may not work as expected because the project is now in an unpredictable state (think of this like running `yarn` after manually modifying your **node_modules** directory).
+<BoxLink
+  title="Expo Modules API: Overview"
+  description="An overview of the APIs and utilities provided by Expo to develop native modules."
+  href="/modules/overview/"
+  Icon={BookOpen02Icon}
+/>
 
-If you want to make static changes to your native project files, such as **AndroidManifest.xml** or **Info.plist**, and still have access to prebuilding, then create a [config plugin](/config-plugins/plugins-and-mods/#create-a-plugin) to see how you can hook into the prebuild process to make those changes.
+<BoxLink
+  title="Tutorial: Creating a native module"
+  description="A tutorial on creating a native module that persists settings with the Expo Modules API."
+  href="/modules/native-module-tutorial/"
+  Icon={BookOpen02Icon}
+/>
 
-### Revert changes from `npx expo run:[android|ios]`
+<BoxLink
+  title="Tutorial: Creating a native view"
+  description="A tutorial on creating a native view that renders a native WebView component with the Expo Modules API."
+  href="/modules/native-view-tutorial/"
+  Icon={BookOpen02Icon}
+/>
 
-Suppose you have decided to roll your app back to being fully managed (no **android** or **ios** directories in your project directory). In that case, you can check out your most recent commit before executing `npx expo run:[android|ios]`, then run `npm install` again to restore the state of your **node_modules** directory.
+### Creating a local module
 
-### Develop apps with custom native code
+If you intend to use your native module in a single app (you can always change your mind later), we recommend [using a "local" Expo module](/modules/get-started/#creating-the-local-expo-module) to write custom native code. Local Expo Modules function similarly to [Expo Modules](/modules/overview) used by library developers and within the Expo SDK, like `expo-camera`, but they are not published on npm. Instead, you create them directly inside your project.
 
-Once you have customized the native code in your project, you can use the [`expo-dev-client`](/develop/development-builds/introduction/#what-is-an-expo-dev-client) library to [create a development build](/develop/development-builds/create-a-build/) and retain the convenience of working with JavaScript or TypeScript in Expo Go.
+Creating a local module scaffolds a Swift and Kotlin module inside the `modules` directory in your project, and these modules are automatically linked to your app.
 
-## Release apps with custom native code to production
+<Terminal cmd={['npx create-expo-module@latest --local', 'npx expo run']} />
```

Comment:
```suggestion
<Terminal cmd={['$ npx create-expo-module@latest --local', '$ npx expo run']} />
```

We need to format these commands with a `$` sign so they are represented in the command format.